pages/use-solid.md

```diff
@@ -35,15 +35,22 @@ The following table lists some Pod Provider:
 
 | Provider                                           | Responsible for Domain Name and Terms                                      | Responsible for Hosting                       | Hosting Location | Solid Implementation                                        |
 |----------------------------------------------------|----------------------------------------------------------------------------|-----------------------------------------------|------------------|-------------------------------------------------------------|
-| [Inrupt Pod Spaces](https://start.inrupt.com/) | [Inrupt, Inc.](https://inrupt.com/terms-of-service)                        | [Amazon](https://aws.amazon.com)              | Germany              | [ESS](https://inrupt.com/products/enterprise-solid-server/) |
+| [Inrupt Pod Spaces](https://start.inrupt.com/) | [Inrupt, Inc.](https://www.inrupt.com/terms-conditions)                        | [Amazon](https://aws.amazon.com)              | Germany              | [ESS](https://inrupt.com/products/enterprise-solid-server/) |
+| [inrupt.net](https://inrupt.net/) | [Inrupt, Inc.](https://www.inrupt.com)                                                      | [Amazon](https://aws.amazon.com)              | USA              | [NSS](https://github.com/solid/node-solid-server)           |
 | [solidcommunity.net](https://solidcommunity.net/)  | [Solid Project](https://github.com/solid/solidcommunity.net_operations)    | [Digital Ocean](https://www.digitalocean.com) | UK               | [NSS](https://github.com/solid/node-solid-server)           |
 | [solidweb.org](https://solidweb.org)               | [Solid Grassroots](https://gitlab.com/groups/solidweb.org/-/group_members) | [Hosteurope](https://www.hosteurope.de)       | France          | [NSS](https://github.com/solid/node-solid-server)           |
```

Comment:
>We apologize for the inconvenience arising through this security [issue](https://github.com/nodeSolidServer/node-solid-server/issues/1768).
>Consider using [solidcommunity.net](https://solidcommunity.net/).

The current corsProxy feature is essentially missing authentication and/or moderation of some sort based on service policy. Simply don't enable corsProxy on the service.

As of this writing, solidcommunity.net offers corsProxy, so the recommendation to "consider using" it doesn't address the "issue".